.github/workflows/tests.yml

```diff
@@ -34,13 +34,14 @@ jobs:
 
     - uses: actions/checkout@v3
 
-    - name: Setup Python
-      uses: ./.github/actions/python_cache/
+    - uses: actions/setup-python@v4
+      with:
+        python-version: "3.8"
 
-    - name: Install Haystack
+    - name: Install Black
       run: |
         pip install --upgrade pip
-        pip install .[dev]
+        pip install black==22.6.0
```

Comment:
Just wondering here: how could we keep in sync this version, the `dev` version, and the pre-commit hooks version? I imagine black to be fairly stable, so not a big issue right now. It's just a "theoretical" problem that maybe we'll need to address in the future

Reply:
Probably we could have a `[linting]` or `[formatting]` section in `pyproject.toml` that is installed with `[dev]` too and install only that. 

But as you said it's pretty stable, I wouldn't be much concerned by this.